top.php

```diff
@@ -316,4 +316,4 @@
 	}
 }
 require_once(FOOTERF);
-?>
\ No newline at end of file
+?>
```

Comment:
There is now an extraneous newline after the closing `?>` tag.  Please remove the `?>` entirely to prevent the possibility of unexpected HTML getting sent after the page is processed.